at_onboarding_flutter/lib/screen/at_onboarding_backup_screen.dart

```diff
@@ -105,7 +115,7 @@ class _AtOnboardingBackupScreenState extends State<AtOnboardingBackupScreen> {
                 height: 48,
                 borderRadius: 24,
                 child: Text(
-                  AtOnboardingStrings.saveButtonTitle,
+                  "Backup Now",
```

Comment:
@sonle-geekyants Can we use String constants here ?

Reply:
I've fixed